CHANGELOG.md

```diff
@@ -1,10 +1,28 @@
 # Changelog
 
+## [6.0.0](https://github.com/theforeman/puppet-pulpcore/tree/6.0.0) (2022-04-19)
+
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/5.2.1...6.0.0)
+
+**Breaking changes:**
+
+- Drop Pulpcore 3.14 & 3.15, move to 3.16 and 3.17 [\#249](https://github.com/theforeman/puppet-pulpcore/pull/249) ([ekohl](https://github.com/ekohl))
+
+**Implemented enhancements:**
+
+- Fixes [\#34684](https://projects.theforeman.org/issues/34684) - install pulp-cli [\#252](https://github.com/theforeman/puppet-pulpcore/pull/252) ([evgeni](https://github.com/evgeni))
+- Allow extlib 6.x, apache 7.x, stdlib 8.x, postgresql 8.x [\#246](https://github.com/theforeman/puppet-pulpcore/pull/246) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Disable default Apache mods in acceptance spec [\#254](https://github.com/theforeman/puppet-pulpcore/pull/254) ([wbclark](https://github.com/wbclark))
+- Generate the acceptance test CA with SAN [\#253](https://github.com/theforeman/puppet-pulpcore/pull/253) ([ekohl](https://github.com/ekohl))
```

Comment:
I added `skip-changelog` to both of these, if you don;t mind regenerating the changelog this section will go away.